boards/beagle/beaglebone_ai64/doc/index.rst

```diff
@@ -0,0 +1,105 @@
+.. _beaglebone_ai64:
+
+BeagleBone AI-64
+################
+
+Overview
+********
+BeagleBone AI-64 is a computational platform powered by TI J721E SoC, which is
+targeted for automotive applications.
+
+.. figure:: assets/bbai_64.webp
+		:align: center
+		:width: 600px
+		:alt: BeagleBoard.org BeagleBone AI-64
+
+Hardware
+********
+The BeagleBone AI-64 is powered by TI J721E SoC, which has three domains (Main,
+MCU, WKUP). This document gives overview of Zephyr running on Cortex R5 in the
+Main domain.
+
+L1 Memory System
+----------------
+* 16 KB instruction cache.
+* 16 KB data cache.
+* 64 KB TCM.
+
+Region Address Translation
```

Comment:
Q: Other k3 SoCs configure RAT in soc.c, is that required here?

Reply:
It can be used, but is not required. For now we use the default RAT mapping.